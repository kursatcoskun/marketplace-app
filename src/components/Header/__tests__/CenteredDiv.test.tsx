// MyComponent.test.js
import React from "react";
import { shallow } from "enzyme";
import { CenteredDiv } from "../CenteredDiv";
describe("CenteredDivComponent", () => {
  it("should render CenteredDiv component", () => {
    const wrapper = shallow(<CenteredDiv />);
  });

  it("should render initial layout", () => {
    const component = shallow(<CenteredDiv />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
