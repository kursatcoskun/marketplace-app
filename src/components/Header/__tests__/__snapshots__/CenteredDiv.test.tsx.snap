// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenteredDivComponent should render initial layout 1`] = `
Array [
  <div
    className="sc-bdnxRM kuDnwP"
  />,
]
`;
